bftengine/include/bcstatetransfer/SimpleBCStateTransfer.hpp

```diff
@@ -40,7 +42,7 @@ namespace SimpleBlockchainStateTransfer {
 // The application/storage layer is responsible to store the digests in the
 // blocks.
 // Blocks are numbered. The first block should be block number 1.
-constexpr static uint32_t BLOCK_DIGEST_SIZE = 32;
+inline constexpr auto BLOCK_DIGEST_SIZE = concord::storage::blockchain::BLOCK_DIGEST_SIZE;
```

Comment:
That's a good point and I agree. I was thinking of something like a trait template that users can specialise  and specify the constant that way. Something like `std::hash()`. However, I was looking at `bftengine`, `kvbc` and `storage` directories and I am not sure yet where to put the current specialisation with 32. It seems like it should be in `kvbc`, but need to think it through.

In the meantime, I have done further refactoring and will push that first.

Reply:
Actually, I am reverting that part as it is the state transfer code that computes block hashes and determines the size of 32 bytes. In doing so, the dependency stays from state transfer to a specific implementation in block.h/cpp .

I am thinking this can be resolved at a later stage.